src/main/java/Task.java

```diff
@@ -1,10 +1,15 @@
 public class Task {
     protected String description;
+    protected String taskType;
     protected boolean isDone;
+    protected String eventWhen;
+
 
     public Task(String description) {
         this.description = description;
         this.isDone = false;
+        this.taskType = "";
+        this.eventWhen = "";
```

Comment:
Maybe can make the eventWhen sound more noun like eventDate.

Reply:
Right, that makes more sense :') thanks for the suggestion!